proposals/description/tip_118.ts

```diff
@@ -0,0 +1,247 @@
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.ethPSM, // pcvDeposit
+        addresses.fei, // token
+        addresses.daiFixedPricePSM, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Migrate FEI from ETH PSM to the DAI PSM'
+    },
+
+    // LUSD_PSM
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.lusdPSM, // pcvDeposit
+        addresses.lusd, // token
+        addresses.lusdHoldingDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Migrate LUSD from the LUSD PSM to the LUSD holding PCV deposit'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.lusdPSM, // pcvDeposit
+        addresses.fei, // token
+        addresses.daiFixedPricePSM, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Migrate FEI from the LUSD PSM to the DAI PSM'
+    },
+
+    // 2. Revoke PSM permissions
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('MINTER_ROLE'), addresses.ethPSM],
+      description: 'Revoke MINTER_ROLE from ETH PSM'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('MINTER_ROLE'), addresses.lusdPSM],
+      description: 'Revoke MINTER_ROLE from LUSD PSM'
+    },
+
+    // RAI PSM does not have MINTER_ROLE
+
+    // 3. Pause all unpaused functionality on the PSMs
+    {
+      target: 'raiPriceBoundPSM',
+      values: '0',
+      method: 'pauseRedeem()',
+      arguments: (addresses) => [],
+      description: 'Pause redemptions on the Rai Price bound PSM'
+    },
+    {
+      target: 'ethPSM',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: 'Pause the ETH PSM and prevent minting'
+    },
+
+    // 4. Unset deprecated PSMs as safe addresses
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'unsetSafeAddresses(address[])',
+      arguments: (addresses) => [[addresses.ethPSM, addresses.lusdPSM, addresses.raiPriceBoundPSM]],
+      description: 'Unset the ETH, LUSD and RAI PSMs as safe addresses'
+    },
+
+    // Update Collaterization Oracle and set safe addresses
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposits(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.wethHoldingDeposit,
+          addresses.lusdHoldingDeposit,
+          addresses.daiHoldingDeposit,
+          addresses.voltHoldingDeposit
```

Comment:
Not adding `gOHM` as requires an oracle. Can ship the oracle as a seperate PR at a later date if needed